src/components/project/project.resolver.ts

```diff
@@ -312,6 +317,30 @@ export class ProjectResolver {
     );
   }
 
+  @ResolveField(() => FinancialReport, { nullable: true })
+  async dueFinancialReport(
+    @AnonSession() session: Session,
+    @Parent() project: Project
+  ): Promise<PeriodicReport | null> {
+    return await this.periodicReportService.getDueReport(
+      project.id,
+      ReportType.Financial,
+      session
+    );
+  }
+
+  @ResolveField(() => NarrativeReport, { nullable: true })
+  async dueNarrativeReport(
+    @AnonSession() session: Session,
+    @Parent() project: Project
+  ): Promise<PeriodicReport | null> {
+    return await this.periodicReportService.getDueReport(
+      project.id,
+      ReportType.Narrative,
+      session
+    );
+  }
```

Comment:
These resolvers response should be SecuredPeriodicReport.. right?

Reply:
Correct

Reply:
kk